src/robot/writer/rowsplitter.py

```diff
@@ -75,7 +75,10 @@ def _split_row(self, row, indent):
             current, row = self._split(row)
             yield self._escape_last_cell_if_empty(current)
             if row:
-                row = self._continue_row(row, indent)
+                if self._comment_mark in current:
```

Comment:
This will cause the following line to fail, won't it?

`Log      The # of items are ${x}`